src/main/scala/lucuma/broadcastchannel/package.scala

```diff
@@ -13,7 +16,7 @@ package object broadcastchannel {
 
   implicit def ioToOnMessageHandler[T](f: T => IO[Unit]): OnMessageHandler[T] =
     js.ThisFunction.fromFunction2(
-      (_: lucuma.bc.broadcastChannel.broadcastChannelMod.BroadcastChannel[js.Any], x: T) =>
```

Comment:
is `js.Any` replaceable by `Any`?

Reply:
See https://github.com/ScalablyTyped/Converter/issues/351.

Reply:
Cool, thanks